src/EnergyPlus/EMSManager.cc

```diff
@@ -2385,7 +2121,7 @@ void SetupEMSActuator(EnergyPlusData &state,
         actuator.Units = cUnits;
         actuator.Actuated = &lEMSActuated; // Pointer assigment
```

Comment:
Do you notice what is happening here? You are creating a pointer by taking the address of a reference which is a syntactically dereferenced pointer. 📌 I would change this API to take pointers. Then again, if it were up to me, I would just use pointers instead of references for everything.

Reply:
I did not notice. You are creating a pointer (bool *Actuated) by taking the address (&lEMSActuated) of a reference (function argument bool &lEMSActuated) wihch is a ... and that's as far as I got. This (&lEMSActuated) is dereferencing a reference to a bool. I missed that pop quiz.